src/meds/schema.py

```diff
@@ -73,12 +73,22 @@ def patient_schema(per_event_metadata_schema=pa.null()):
         ("patient_id", pa.int64()),
         ("prediction_time", pa.timestamp("us")),
         ("boolean_value", pa.bool_()),
+        ("integer_value", pa.int64()),
+        ("float_value", pa.float32()),
```

Comment:
Can we make this float64 instead? It shouldn't cost us much and will help avoid issues if people have float64 labels.